src/Datadog.Trace/TracingProcessManager.cs

```diff
@@ -218,8 +221,8 @@ private static bool ProgramIsRunning(string fullPath)
                 return false;
             }
 
-            var fileName = Path.GetFileNameWithoutExtension(fullPath);
-            var processesByName = Process.GetProcessesByName(fileName);
+            var processId = GetProcessIdFromFileName(fullPath);
+            var processesByName = Process.GetProcessesByName(processId);
```

Comment:
I'm confused by process Ids being strings instead of integers, especially here in `Process.GetProcessesByName(processId)`. `GetProcessesByName()` expects a process name, not a pid. What's going on here? Is `processId` not actually the process id?

Reply:
Refactor bug, reverting.